site/src/modules/provisioners/ProvisionerGroup.tsx

```diff
@@ -0,0 +1,213 @@
+import { useTheme } from "@emotion/react";
+import Business from "@mui/icons-material/Business";
+import Person from "@mui/icons-material/Person";
+import Button from "@mui/material/Button";
+import Tooltip from "@mui/material/Tooltip";
+import type { BuildInfoResponse, ProvisionerDaemon } from "api/typesGenerated";
+import { DropdownArrow } from "components/DropdownArrow/DropdownArrow";
+import { Pill } from "components/Pill/Pill";
+import { type FC, useState } from "react";
+import { createDayString } from "utils/createDayString";
+import { ProvisionerTag } from "./ProvisionerTag";
+
+type ProvisionerGroupType = "builtin" | "psk" | "key";
```

Comment:
Is PSK a pre-shared key?

Reply:
yeah. I don't love the current terminology, but it at least matches the backend